elevate-cpanel

```diff
@@ -911,12 +913,19 @@ sub capture_imunify_features {
 
     my $output   = Cpanel::SafeRun::Simple::saferunnoerror( IMUNIFY_AGENT, qw{features list} );
     my @features = map { s/\s+//g; $_ } grep { m/\S/ } split( "\n", $output );
+
+    if ( -f IMUNIFY_LICENSE_FILE ) {
+        File::Copy::move( IMUNIFY_LICENSE_FILE, '/var/cpanel' );
```

Comment:
this can be confusion this is not using `IMUNIFY_LICENSE_BACKUP`

Reply:
incomplete conversion. try now.